connection.go

```diff
@@ -671,10 +671,10 @@ type snowflakeArrowStreamChunkDownloader struct {
 }
 
 func (scd *snowflakeArrowStreamChunkDownloader) Location() *time.Location {
-	if scd.sc != nil {
+	if scd.sc != nil && scd.sc.cfg != nil {
```

Comment:
This change is not a mistake, but I'm wondering - you actually encountered this situation? By looking at the code I can see that when `snowflakeConn` is initialised, it is always initialised with not nil config.

Reply:
I have not experienced this, I can remove it I just wanted to ensure that we dont null pointer dereference in any possible way